lib/ui/painting/matrix.cc

```diff
@@ -32,6 +32,23 @@ SkMatrix ToSkMatrix(const tonic::Float64List& matrix4) {
   return sk_matrix;
 }
 
+SkM44 ToSkM44(const tonic::Float64List& matrix4) {
+  FML_DCHECK(matrix4.data());
+  SkM44 sk_m44;
+  for (int i = 0; i < 16; ++i) {
+    sk_m44.setRC(i % 4, i / 4, SafeNarrow(matrix4[i]));
+  }
+  return sk_m44;
+}
+
+tonic::Float64List ToMatrix4FromSkM44(const SkM44& m44) {
+  tonic::Float64List matrix4(Dart_NewTypedData(Dart_TypedData_kFloat64, 16));
+  for (int i = 0; i < 16; ++i) {
+    matrix4[i] = m44.row(i % 4)[i / 4];
```

Comment:
Ditto.

Reply:
This function has no reference and it's unnecessary. I'll delete it :)